### PR TITLE
src: display bot-icon as background image in BotBuilder for user's bot

### LINKED ISSUE #1551: Display bot icon while listing all bots of the user
**Actual Behaviour**

Bot icon not getting displayed in the list private bots section (https://skills.susi.ai/botbuilder)

**Expected Behaviour**

Bot icon should be displayed inside each card. 
Bot icons are given by the skill creator, when the skill is being created:
image
Bot icons can be accessed by using the `getImage.png` API with the following get parameters:
- `access_token`: the access token of the currently logged user
- `language`: the language of the skill
- `group`: group name of the skill
- `image`: image name for the skill

Example: `https://api.susi.ai/cms/getImage.png?access_token=<access_token>&language=en&group=Business%20and%20Finance&image=images/elon%20musk%20car.jpg`

**Screenshots of the issue**
image

The bot icon for each skill card should be displayed on it.

**Would you like to work on the issue?**

No, anyone can take this up. If you have any doubts please ask below.

### PR DESCRIPTION
**Pull request by AdityaSrivast**
Fixes #1551 
Changes: While listing bots of a user (in botbuilder component), the background of the card should display the bot-icon uploaded by the user while creating the bot.

Surge Deployment Link: https://pr-1594-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
Earlier:
image
Now:
image

### CODE REVIEW

**Comment by AdityaSrivast** [comment]
@hkedia321 Can you please review my changes?

File: src/components/BotBuilder/BotBuilder.js [hunk 1]
```diff
@@ -58,45 +59,96 @@ class BotBuilder extends React.Component {
...
+          name +
+          '&private=1&access_token=' +
+          cookies.get('loggedIn');
+        $.ajax({
+          url: url,
+          dataType: 'jsonp',
+          crossDomain: true,
+          success: function(data) {
+            data = data.text.split('\n::');
+            bot.image = data[23].split(' ')[1];
```
Comment (hkedia321) [inline src/components/BotBuilder/BotBuilder.js]: Please use instead `data.text.match(/^::image\s(.*)$/m)` to extract the image name. Its not necessary that the image will be at the `23` index always.

Comment (akshatnitd) [reply src/components/BotBuilder/BotBuilder.js]: @AdityaSrivast Please use the regex pattern as suggested by @hkedia321 .

**Comment by AdityaSrivast** [comment]
@hkedia321 @akshatnitd

**Comment by AdityaSrivast** [comment]
@hkedia321 @akshatnitd changes made. Please review...

**Comment by AdityaSrivast** [comment]
@DNS-404 made the changes. Can you please review?

**Pull request merged**